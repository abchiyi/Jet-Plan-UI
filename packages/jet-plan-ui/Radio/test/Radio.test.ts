import { expect, describe, test } from "vitest";
import { mount } from "@vue/test-utils";
import component from "../src/index.vue";

describe("Test Radio", () => {
  test("SnapShots", () => {
    const wrapper = mount(component);
    expect(wrapper.html()).matchSnapshot();
  });
});
