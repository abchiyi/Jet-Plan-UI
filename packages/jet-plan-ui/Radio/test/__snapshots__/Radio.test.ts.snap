// Vitest Snapshot v1

exports[`Test Radio > SnapShots 1`] = `
"<label class=\\"shape j-radio input-hidden\\"><input><svg viewBox=\\"0 0 16 16\\">
    <rect class=\\"background\\" height=\\"16\\" width=\\"16\\" rx=\\"8\\"></rect>
    <rect transform=\\"translate(2 2)\\" class=\\"mask\\" height=\\"12\\" width=\\"12\\" rx=\\"6\\"></rect>
    <g class=\\"dot\\">
      <rect transform=\\"translate(5 5)\\" height=\\"6\\" width=\\"6\\" rx=\\"3\\"></rect>
    </g>
  </svg></label>"
`;
